Samples/Windowing/cs-wpf/wpf_packaged_app (Package)/wpf_packaged_app (Package).wapproj

```diff
@@ -53,7 +53,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.1.0-preview2" />
```

Comment:
This should be "1.1.0" not targeting the preview2 release.
Note: this is perhaps not blocking for this PR, but needs to be updated once the 1.1.0 release is live.